developers/opencv.md

```diff
@@ -338,8 +334,8 @@ int main(int argc, char *argv[]) {
 
     gchar *descr = g_strdup(
         "udpsrc port=5600 "
-        "! application/x-rtp, payload=96 ! rtph264depay ! h264parse ! avdec_h264 "
```

Comment:
I think it is pretty standard to have a h264parse after depay. do you know why it had to be removed?

Reply:
good catch, take a look now